### ISSUE TITLE
Historical flight cost data

### ISSUE THREAD

**Issue by mrjkey** [Tris20/FairFareFinder#18]
We may not be able to get future costs very easily, but maybe we could build out a pre calculated estimated cost of flights. From Berlin, to city X, which month, which day of the week, are there holidays, etc.

**Comment by Tris20** [comment]
Agree. I think even providing the average cost in general would be helpful and probably somewhat ok to get.
